docs/development/module/tests.md

```diff
@@ -94,7 +94,7 @@ Account 1 has balance 10, account 2 has balance 20, and so on.
 
 It will be useful to simulate block production to verify that expected behavior holds during block time dependent changes. 
 
-A simple way of doing this increments the System module's block number between `on_initialize` and `on_finalize` calls from all modules with `System::block_number()` as the sole input.
+A simple way of doing this increments the System module's block number between `on_initialize` and `on_finalize` calls from all modules with `System::block_number()` as the sole input. While it is important for runtime code to [cache calls](https://substrate.dev/recipes/storage/cache.html) to storage or the system module, the test environment scaffolding should prioritize readability to facilitate future maintenance.
```

Comment:
The UI is making it unintuitive to reply to this https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/254#discussion_r352620401 so I'll just link it.

@JoshOrndorff It's a test so it doesn't really matter how many calls are made to system or storage. Computation/storage efficiency aren't that important in the test environment scaffolding. With that said, it is definitely not how things should be done for runtime development and I wonder if that might be confusing for readers of the doc. So I added this note.